<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Dipjyoti-PC" name="Command line test" tests="4" failures="1" timestamp="5 Sep 2017 15:44:11 GMT" time="7.294" errors="0">
  <testcase name="feature" time="2.055" classname="com.intuit.karate.testng.KarateTestngTest"/>
  <testcase name="feature" time="0.158" classname="com.intuit.karate.testng.KarateTestngTest"/>
  <testcase name="feature" time="0.489" classname="com.intuit.karate.testng.KarateTestngTest"/>
  <testcase name="testParallel" time="3.754" classname="com.intuit.karate.testng.KarateTestngTest">
    <failure type="java.lang.AssertionError" message="expected [true] but found [there are scenario failures]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [there are scenario failures]
	at com.intuit.karate.testng.KarateRunner.testParallel(KarateRunner.java:71)
... Removed 38 stack frames]]>
    </failure>
  </testcase> <!-- testParallel -->
</testsuite> <!-- Command line test -->
